test/integration/dbt_test.py

```diff
@@ -97,7 +97,8 @@ def test_dbt_helper(
         [
             "dbt_helper",
             "add-tables",
-            f"--{dbt_target}",
+            "--target",
+            dbt_target,
```

Comment:
The logic here is simpler and more legible with an arg that explicitly sets the target (full/fast).